src/carbon_alkalinity_nutrients.jl

```diff
@@ -122,10 +122,25 @@ end
 @inline freshwater_virtual_flux() = 0.0
```

Comment:
```suggestion
@inline freshwater_virtual_flux() = 0
```

We should avoid writing "0.0" (which is double precision 0) whenever possible. Typically the integer `0` is safe, since it will be promoted to whatever floating point precision is used in a calculation. If we actually need a floating point, then we need to evaluate it from the input, eg using the function `zero`:

```julia
f(x) = zero(x)
```

Single precision is definitely on our radar, since it's essentially 2x faster (half the carbon footprint for a simulation!)